api/client/lib.py

```diff
@@ -447,6 +447,21 @@ def rank_series_by_source(access_token, api_host, series_list):
             yield series_with_source
 
 
+def get_available_timefrequency(access_token, api_host, **series):
+    def make_key(key):
```

Comment:
would we make this function shared between `get_available_timefrequency` and `get_source_ranking` ?

Reply:
done